c7n/actions.py

```diff
@@ -461,16 +461,35 @@ def send_data_message(self, message):
 
     def send_sns(self, message):
         topic = self.data['transport']['topic']
-        region = topic.split(':', 5)[3]
+        if len(topic.split(':', 5)) == 6:
```

Comment:
from a readability perspective, this would be much better with some letters instead of magic numbers. ie. `if topic.startswith('arn:')`